android/src/main/java/com/tailscale/ipn/ui/view/AboutView.kt

```diff
@@ -71,7 +72,7 @@ fun AboutView(backToSettings: BackNavigation) {
                         Modifier.clickable {
                           localClipboardManager.setText(AnnotatedString(BuildConfig.VERSION_NAME))
```

Comment:
can you add a comment here noting that the clipboard version should be the long form version and the displayed version is the short version please?

Reply:
done, good call